test/colony.js

```diff
@@ -733,4 +735,63 @@ contract("Colony", addresses => {
       await checkErrorRevert(colony.claimPayout(taskId, MANAGER_ROLE, token.address, { from: OTHER }));
     });
   });
+
+  describe("roles and permissions", () => {
```

Comment:
Can you merge this section with `when working with permissions` above? When reading the descriptions they seem the same.